platformio-example.ini

```diff
@@ -28,5 +28,5 @@ lib_deps=
   Adafruit BME680 Library
   Adafruit BME280 Library
   Adafruit Unified Sensor
-  IotWebConf@<3.0.0
+  https://github.com/prampec/IotWebConf.git
   MCCI LoRaWAN LMIC library
```

Comment:
so this works for platformio, but how about arduino ide users?

Reply:
Arduio users should  load the ZIP from Github and install the zip using library manager.
All this is necessary only as long as there is no official release!

Reply:
the "release" situation is a mess.

otoh, author says it is not officially released, otoh, platformio automatically pulls the 3.0 version already (if not held back by adding `<3.0`).

that arduino users have to manually download and install a zip is also not very convenient.

so I guess we should wait with merging this until everybody gets the new officially really-really-released version.

Reply:
I totally agree. Let's wait.

Reply:
https://github.com/prampec/IotWebConf link to check whether 3.0 is officially released.

Reply:
v3.0.1 is now really really officially released! \o/

but @rexfue told at last meeting that this PR needs some more changes to work ok with 3.0.1, so let's wait until these changes are done.